File: 3.2 Complete Search/Iterative 2D Nested/p1260.cpp
// Competitive Programming 3
// Problem 927

#include <iostream>

using namespace std;

int main(){
    int T;
    cin >> T;

    while(T--){
        int N, count = 0;
        cin >> N;
        int A[N];

        for (int i = 0; i < N; i++)
            cin >> A[i];

        for (int i = 1; i < N; i++){
            for (int j = 0; j < i; j++){
                if (A[i] >= A[j])
                    count += 1;
            }
        }
        cout << count << endl;
    }
    return 0;
}

File: 3.2 Complete Search/Iterative 2D Nested/p1260.py
# Competitive Programming 3
# Problem 927

T = int(input())

for _ in range(T):
    N = int(input())
    A = [int(x) for x in input().split()]
    
    count = 0
    
    for i in range(1,N):
        for j in range(i):
            count += 1 if A[i] >= A[j] else 0
            
    print(count)
            
    